### ISSUE TITLE
Still getting ads on twitch while using TwitchAdSolutions (vaft) userscript

### ISSUE THREAD

**Issue by Jeklund6** [pixeltris/TwitchAdSolutions#225]
I previously used the ublock filter, then the permalink version, now using userscript as recommended in the readme, still getting twitch ads. I also removed the old filter and unset the resources while using the userscript, no difference. Haven't seen anyone else with this issue, am I doing something wrong? Appreciate any feedback.

**Comment by Jeklund6** [comment]
![image](https://github.com/pixeltris/TwitchAdSolutions/assets/34612810/d031e06b-0a03-4032-aba7-8b86cc91e9fe)

**Comment by Nikralx** [comment]
Personally I had issues with the vaft long ago and switched. I am using the Video-swap-new Userscript on FF and I am not seeing any ads. Have you tried that one?

**Comment by MarcDwyer** [comment]
I don't think you're doing anything wrong. It's not working for me either. Looking at the script it seems like its having trouble identifying when an m3u8 is an ad. The AD_SIGNIFIER variable no longer seems to be a reliable way to identify ads.

**Comment by kisiel92** [comment]
Both vaft and video-swap-new appear to be completely broken as of today using Userscripts on Safari

**Comment by OnePunkArmy** [comment]
When I use the vaft userscript, streams just never load at all. When I use the vaft uBlock, ads still get through. Seems like Twitch is currently winning the war.

**Comment by level3tjg** [comment]
> Both vaft and video-swap-new appear to be completely broken as of today using Userscripts on Safari

These scripts never worked on Safari due to the way WebKit loads userscripts. WebKit loads userscripts after initial page load unlike other implementations, it's determined at compile time so there's nothing you can really do about it. If you're on macOS you'll need to use another browser

**Comment by kisiel92** [comment]
> > Both vaft and video-swap-new appear to be completely broken as of today using Userscripts on Safari
> 
> These scripts never worked on Safari due to the way WebKit loads userscripts. WebKit loads userscripts after initial page load unlike other implementations, it's determined at compile time so there's nothing you can really do about it. If you're on macOS you'll need to use another browser

Most of the time they do work on Safari, actually. I haven't had any issues with them blocking ads on Safari the past week or two. On the off chance an ad does come through, a reload tends to do the trick

**Comment by level3tjg** [comment]
> Most of the time they do work on Safari, actually. I haven't had any issues with them blocking ads on Safari the past week or two. On the off chance an ad does come through, a reload tends to do the trick

Wow you're right, it's working on iOS and macOS now. I tried ~6 months ago and the script execution order was causing it to fail, something must have changed since last time I tried.

**Comment by salvadorzombie** [comment]
Both of the scripts (vaft and video-swap-new) stopped working for me last night. I've been using video-swap-new and at some time between 11pm and 1am (best guess) I first got an ad on Twitch. Usually when one gives me trouble (maybe every 4-5 months) I swap to the other and it works fine. Neither was working until a few hours ago, when video-swap-new suddenly started blocking again. Just now about five minutes ago, I got another ad while using video-swap-new, and swapping to vaft isn't working either).
